yttbtc/cli.py

```diff
@@ -7,22 +7,85 @@
 from typing import Optional
 import typer
 from requests_html import HTMLSession
-from youtube_transcript_api import YouTubeTranscriptApi
-import os
-
 from yttbtc import __app_name__, __version__
 import subprocess
+import pytube
+from moviepy.editor import VideoFileClip
+import pywhisper
+import os
+import static_ffmpeg
 
 app = typer.Typer()
 
 session = HTMLSession()
+os.environ["IMAGEIO_FFMPEG_EXE"] = "/usr/bin/ffmpeg"
+
+def download_video(url):
+    video = pytube.YouTube(url)
+    stream = video.streams.get_by_itag(18)
+    stream.download()
+    return stream.default_filename
+
+
+def convert_to_mp3(filename):
+    clip = VideoFileClip(filename)
+    clip.audio.write_audiofile(filename[:-4] + ".mp3")
+    clip.close()
+
+
+def AudiotoText(filename):
+    model = pywhisper.load_model("base")
```

Comment:
I'd maybe start with the tiny model and would certainly offer this as an option to toggle in the future.